device-types/Cisco/C9800-L-F.yaml

```diff
@@ -0,0 +1,36 @@
+---
+manufacturer: Cisco
+model: Catalyst 9800-L-F
+slug: cisco-c9800-l-f
+part_number: C9800-L-F
+u_height: 1
+is_full_depth: false
+comments: '[Cisco Catalyst 9800-L Wireless Controller Data Sheet](https://www.cisco.com/c/en/us/products/collateral/wireless/catalyst-9800-series-wireless-controllers/datasheet-c78-742434.html)'
+console-ports:
+  - name: Con
+    type: rj-45
+  - name: usb0
+    type: usb-micro-b
+interfaces:
+  - name: Service Port
+    type: 1000base-t
+    mgmt_only: true
+  - name: Redundancy Port
+    type: 1000base-t
+    mgmt_only: true
+  - name: TenGigabitEthernet1
+    type: 10gbase-x-sfpp
+  - name: TenGigabitEthernet2
+    type: 10gbase-x-sfpp
+  - name: GigabitEthernet1
+    type: 2.5gbase-t
+  - name: GigabitEthernet2
+    type: 2.5gbase-t
+  - name: GigabitEthernet3
+    type: 2.5gbase-t
+  - name: GigabitEthernet4
+    type: 2.5gbase-t
+power-ports:
+  - name: PS1
+    type: iec-60320-c14
```

Comment:
According to the manufacturer specification, this is a dc-terminal with an external AC to DC adapter. Please change this to `dc-terminal`